scripts/zones/East_Ronfaure/TextIDs.lua

```diff
@@ -13,7 +13,7 @@ LOGGING_IS_POSSIBLE_HERE = 7482; -- Logging is possible here if you have
 -- General Dialog
      RAYOCHINDOT_DIALOG = 7406; -- If you are outmatched, run to the city as quickly as you can.
      CROTEILLARD_DIALOG = 7407; -- Sorry, no chatting while I'm on duty.
-      BLESSED_WATERSKIN = 7451; -- To get water, rade the waterskin you hold with the river. 
+      BLESSED_WATERSKIN = 7451; -- To get water, radethe waterskin you hold with the river.
```

Comment:
That's what I initially changed it to, but the tool wouldn't pick up on it. Probably due to the curly quotes around trade. So I plopped in what it spit out in the filtered table, reran, and it latched onto it that time.

Text field from POLUtils:
```
To get water, “trade” the waterskin you hold with the river.≺Prompt≻
```

filtered-dialog-table-101.xml
```
<thing type="DialogTableEntry"><field name="index">7451</field><field name="text">To get water, radethe waterskin you hold with the river. </field></thing>
```